elements/core/components/Tag/Tag.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import {
+  TagWrapper,
+} from './Tag.styles';
+
+interface TagProps {
+  name: string
+  icon?: string,
+}
+/* eslint-disable react/no-danger */
+const Tag = ({
+  name,
+  icon,
+}: TagProps) => (
+  <TagWrapper data-testid="tag">
+    {icon ? <div dangerouslySetInnerHTML={{ __html: icon }} /> : name}
```

Comment:
Icon should only be path to svg on the server, not svg tag, please also change the corresponding tests